internal/serverinstall/nomad.go

```diff
@@ -14,9 +14,10 @@ import (
 )
 
 var (
-	nomadRegionF      string
-	nomadDatacentersF []string
-	nomadNamespaceF   string
+	nomadRegionF         string
+	nomadDatacentersF    []string
+	nomadNamespaceF      string
+	nomadPolicyOverrideF bool
```

Comment:
You'll need to register the flag with the cli framework for it to be available. That can be done in the `NomadFlags` function at the end of this file.

Reply:
🤦 I knew I missed something.  Thank you.

Reply:
@nickethier let me know if that's good